src/main/java/com/monumental/models/Monument.java

```diff
@@ -74,13 +74,18 @@ public Monument() {
         this.contributions = new ArrayList<>();
     }
 
-    public Monument(String artist, String title, Date date, String city, String state) {
+    public Monument(String artist, String title, Date date, String city, String state, String address,
```

Comment:
we should probably just ditch this whole constructor honestly and just use the setters

Reply:
Coming in the next commit.